udata/core/dataset/rdf.py

```diff
@@ -339,36 +339,68 @@ def contact_point_from_rdf(rdf, dataset):
 
 
 def spatial_from_rdf(graph):
+    geojsons = []
     for term in graph.objects(DCT.spatial):
-        for object in term.objects():
-            if isinstance(object, Literal):
-                if object.datatype.__str__() == 'https://www.iana.org/assignments/media-types/application/vnd.geo+json':
-                    try:
-                        geojson = json.loads(object.toPython())
-                    except ValueError as e:
-                        log.warning(f"Invalid JSON in spatial GeoJSON {object.toPython()} {e}")
-                        continue
-                elif object.datatype.__str__() == 'http://www.opengis.net/rdf#wktLiteral':
-                    try:
-                        # .upper() si here because geomet doesn't support Polygon but only POLYGON
-                        geojson = wkt.loads(object.toPython().strip().upper())
-                    except ValueError as e:
-                        log.warning(f"Invalid JSON in spatial WKT {object.toPython()} {e}")
+        try:
+            # This may not be official in the norm but some ArcGis return 
+            # bbox as literal directly in DCT.spatial.
+            if isinstance(term, Literal):
+                bbox = term.toPython().strip().split(',')
```

Comment:
Maybe you could move this is in a dedicated function? :pray:

Reply:
Done :-)